src/components/pages/Save/saveMigration/types.ts

```diff
@@ -0,0 +1,21 @@
+import { BigDecimal } from '../../../../web3/BigDecimal';
+
+export interface StepProps {
+  title: string;
+  buttonTitle?: string;
+  key: string;
+  isCompleted: boolean;
+  isPending: boolean;
+  isActive: boolean;
+  onClick(): void;
+}
+
+export interface State {
```

Comment:
Seems like this could just be 

`type State = StepProps[]` - no benefit to having an object